src/integration-test/tests/citizen/claimantResponse/full_admission_test.ts

```diff
@@ -0,0 +1,136 @@
+import I = CodeceptJS.I
+import { PartyType } from 'integration-test/data/party-type'
+import { UserSteps } from 'integration-test/tests/citizen/home/steps/user'
+import { ClaimantResponseSteps } from 'integration-test/tests/citizen/claimantResponse/steps/claimant-reponse'
+import { Helper } from 'integration-test/tests/citizen/endToEnd/steps/helper'
+import { PaymentOption } from 'integration-test/data/payment-option'
+import { ClaimantConfirmation } from 'integration-test/tests/citizen/claimantResponse/pages/claimant-confirmation'
+import { ClaimantCheckAndSendPage } from 'integration-test/tests/citizen/claimantResponse/pages/claimant-check-and-send'
+import { EndToEndTestData } from 'integration-test/tests/citizen/endToEnd/data/EndToEndTestData'
+import { ClaimantResponseTestData } from './data/ClaimantResponseTestData'
+
+const helperSteps: Helper = new Helper()
+const userSteps: UserSteps = new UserSteps()
+const claimantResponseSteps: ClaimantResponseSteps = new ClaimantResponseSteps()
+const checkAndSendPage: ClaimantCheckAndSendPage = new ClaimantCheckAndSendPage()
+const confirmationPage: ClaimantConfirmation = new ClaimantConfirmation()
+
+Feature('Claimant Response').retry(3)
+
+Scenario('I can as a claimant view the defendants full admission with immediate payment @citizen @admissions', async (I: I) => {
```

Comment:
should this have the citizen tag? would it not be better to just have admissions one and update the command that runs the tests?

Reply:
if we tag them all citizen - we can at least run them all with one tag. then adding more tags to filter from that. i did more 'refactoring' than i thought i was going to do as i thought we would have session to discuss how we wanted to architect the e2e. we should talk about tagging too.